### PR TITLE
No default commands

### PR DESCRIPTION
**Pull request by jacklul**
Do not load example user commands by default.
Reverted versions on 'dummy' user commands back to 1.0.0.

Unit Tests need a fix.

### CODE REVIEW

**Comment by noplanman** [comment]
Tests are failing because there is no `EchoCommand` or `HelpCommand` any more.
As a next step to this PR, as soon as we've moved to an organisation (prod @akalongman), we should move all example commands to a dedicated Repo, no need to have dummy code in the core repo.
Will PR a fix 👍

**Comment by jacklul** [comment]
Yeah, having a repository with example bot will be ideal

**Comment by noplanman** [comment]
> Yeah, having a repository with example bot will be ideal

And on top of that, a repo filled with commands that can be reused.

**Comment by lichtscheu** [comment]
+1 for Organisation!

**Comment by akalongman** [comment]
Done: https://github.com/php-telegram-bot/library

File: tests/unit/Commands/CommandTestCase.php [hunk 1]
```diff
@@ -37,7 +37,8 @@ class CommandTestCase extends TestCase
      */
     public function setUp()
     {
-        $this->telegram = new Telegram('apikey', 'testbot');
+        $this->telegram = new Telegram(self::$dummy_api_key, 'testbot');
+        $this->telegram->addCommandsPath(BASE_COMMANDS_PATH . '/UserCommands');
```
Comment (noplanman) [inline tests/unit/Commands/CommandTestCase.php]: This can be deleted, as there is no more `UserCommands` folder.

**Pull request merged**